packages/react-art/npm/Circle.js

```diff
@@ -17,31 +17,31 @@
 
 'use strict';
 
-var assign = require('object-assign');
-var PropTypes = require('prop-types');
-var React = require('react');
-var ReactART = require('react-art');
+const assign = require('object-assign');
```

Comment:
We can't do this. This file ships on npm without transpilation. So it needs to be ES5. Same for all other files in `/npm/` folder.

Reply:
Ok, I'll change it.

Reply:
changed in 4446de5